src/openrct2/peep/peep.c

```diff
@@ -12932,3 +12918,17 @@ void decrement_guests_heading_for_park(){
         log_error("Attempt to decrement guests heading for park below zero.");
     }
 }
+
+static void peep_release_balloon(rct_peep *peep) {
+    if (peep->item_standard_flags & PEEP_ITEM_BALLOON)
+    {
+        peep->item_standard_flags &= ~PEEP_ITEM_BALLOON;
+
+        if (peep->sprite_type == PEEP_SPRITE_TYPE_BALLOON && peep->x != MAP_LOCATION_NULL)
+        {
+            create_balloon(peep->x, peep->y, peep->z + 10, peep->balloon_colour, false);
```

Comment:
That's very good spot @Gymnasiast! If it's not a parameter that is kept, network version change will be required.